### DISCUSSION TITLE
workrave data question

### DISCUSSION THREAD

**Discussion by carol-GRI** [rcaelers/workrave#378] [General]
I'm responsible for managing computers at my organization and a colleague would like to use Workrave on an organizational machine. Can you tell me what kinds of permissions this tool requires to work and what data, if any, Workrave creates, gathers, and/or communicates from devices where it installed? Thank you!

**Comment by rcaelers** [comment]
No special permissions are required for running Workrave. During installation, the installer requires admin rights to register a DLL for the DeskBand applet. But Workrave is still functional if this step is skipped.

Workrave stores the state of the timers and the statistics you can view in the UI. This information is stored locally. It is not communicated to other computers.

When the networking feature is used, Workrave  communicates its state to connected computers (on the local network). When this feature is used, some additional information is stored (history of the activity monitor, so when/how long the user was active).

Does this answer your question?
